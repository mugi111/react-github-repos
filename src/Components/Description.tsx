import * as React from "react";
import { DescProps } from "../Types";
import "../Styles/Description.scss";

const Description: React.FC<DescProps> = (props: DescProps) => {
  return <p className="github-repos-desc">{props.body}</p>;
};

export default Description;
