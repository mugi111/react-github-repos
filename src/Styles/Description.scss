.github-repos-desc {
  flex: 1 0 auto;
  display: block;
  font-size: 12px;
  margin-bottom: 16px;
  margin-top: 8px;
  color: #586069;
}
