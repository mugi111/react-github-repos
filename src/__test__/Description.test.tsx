import React from "react";
import { shallow } from "enzyme";
import Description from "../Components/Description";

test('Description', () => {
  const component = shallow(<Description body="test description" />)

  expect(component.text()).toEqual('test description');

  expect(component).toMatchSnapshot();
})