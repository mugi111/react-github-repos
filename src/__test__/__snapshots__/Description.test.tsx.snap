// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description 1`] = `
<p
  className="github-repos-desc"
>
  test description
</p>
`;
